ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -121,4 +129,36 @@ boundary server -config="${this.configFilePath || '<path>'}/pki-worker.hcl"`;
       styleActiveLine: false,
     };
   }
+
+  //=methods
+
+  convertCommaSeparatedValuesToArray(input) {
+    return input
+      .split(',')
+      .filter((value) => value.trim())
+      .map((value) => `"${value?.trim()}"`)
+      .join(', ');
+  }
+
+  getTagConfigString() {
+    return this.workerTags
+      .map((tag) => {
+        return `${tag.key} = [${this.convertCommaSeparatedValuesToArray(
```

Comment:
Perhaps this return keyword and brackets is not necessary since an arrow function expression is being used ? Could be changed to just 
.map((tag) => `${tag.key} = [${this.convertCommaSeparatedValuesToArray(tag.value)}]`)

Reply:
Good call, updated it!